src/components/fields/NumberField.js

```diff
@@ -79,6 +82,21 @@ class NumberField extends AskField {
     return { text: this.state.value.length ? this.state.value : '' }
   }
 
+  getHelpMessage () {
+    var helpMessage = this.props.placeholder
+    if (this.props.minValue && this.props.maxValue) {
```

Comment:
what happens if one of these is a negative number? (I would think that someone typing in a negative number here is purposely trying to break it, so probably not a big deal)